splash/qtrender_lua.py

```diff
@@ -449,10 +449,37 @@ def set_custom_headers(self, headers):
         self.tab.set_custom_headers(self.lua2python(headers, max_depth=3))
 
     @command()
+    def get_viewport_size(self):
+        sz = self.tab.web_page.viewportSize()
+        return {'width': sz.width(), 'height': sz.height()}
+
+    @command(table_argument=True)
```

Comment:
Does the following work?
```
@command()
def set_viewport(self, width, height): 
    # ...
```
If it works, then it is better because it would allow both positional and named arguments. "Named arguments" in Lua is just passing a single table as an argument, and this is what we do anyways.